src/plugins/charts/public/static/components/empty_placeholder.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { EuiIcon, EuiText, IconType, EuiSpacer } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+export const EmptyPlaceholder = (props: { icon: IconType }) => (
+  <>
+    <EuiText className="heatmap-chart__empty" textAlign="center" color="subdued" size="xs">
+      <EuiIcon type={props.icon} color="subdued" size="l" />
+      <EuiSpacer size="s" />
+      <p>
+        <FormattedMessage id="charts.noDataLabel" defaultMessage="No results found" />
+      </p>
+    </EuiText>
+  </>
```

Comment:
Would it be better to utilize an `EuiEmptyPrompt` component here? Or would that deviate from how such messages are delivered elsewhere in Visualize?

Reply:
I haven't made any change on this component. It is the component that is used in Lens, I just moved it to the charts plugin in order to be accessible by other plugins
![image](https://user-images.githubusercontent.com/17003240/142992838-c7d753d3-32cb-4dad-b497-4a5a45bd7de0.png)
If I use `EuiEmptyPrompt ` this will also affect Lens (not only Visualize) so it will be better to not do this in this PR. Wdyt?

Reply:
Yeah, if this pattern is used elsewhere, let's keep it as is for now. We can re-evaluate in a more holistic fashion later.